client/actions/schemas/characterSchema.js

```diff
@@ -0,0 +1,11 @@
+import {buildSchema} from 'graphql';
```

Comment:
this file will be duplicated for the backend, therefore we will need to move to this top level when I start on the backend

Reply:
awesome